howto/manage-juju-controllers.md

```diff
@@ -39,10 +39,14 @@ For this how-to you will need the following:
 
 ### Prelude
 
-In order for a Juju controller to trust a JIMM controller, the `login-token-refresh-url` config option must be set to a specific URL path that serves JIMM's public key, which is used to verify signed
-requests when they reach the Juju controller.
+There are 2 ways to add controllers to JAAS/JIMM:
```

Comment:
```suggestion
Adding a controller to JIMM consists of two steps: bootstrap the controller with the `login-token-refresh-url` controller configuration option, then register it with JIMM. There are two ways you can do that: using `juju` and `juju jaas`, or using `jaas` alone.
```

Reply:
Going to skip this one and leave it for the refactoring that you mentioned.